packages/next/src/build/webpack-config-rules/resolve.ts

```diff
@@ -6,22 +6,18 @@ import {
 // exports.<conditionName>
 export const edgeConditionNames = [
   'edge-light',
-  'worker',
```

Comment:
This was originally used as a main field and then moved in https://github.com/vercel/next.js/commit/a035224dc3c1232d75d18d2d40645cc2d56c9ddc#diff-5f441142eb8ba9bcba515aa36fbc420a113307bd191e54a4e9e6c43529fc2613 to a condition. `worker` is a known main field but `edge-light` probably never was? 

I also removed the usage of `edgeConditionNames` as `mainFields` because it's unused and they're different concepts anyway.